media/js/helpfulness.js

```diff
@@ -0,0 +1,91 @@
+(function(win, doc, $) {
+
+    // this feature requires localStorage
+    if (('localStorage' in win)) {
+        var ignore = localStorage.getItem('helpful-ignore') === 'true'; // true if ever clicked ignore
+        var asked_recently = parseInt(localStorage.getItem(doc.location + '#answered-helpful'), 10) > Date.now();
+        if (!ignore && !asked_recently) {
+            // ask about helpfulness after 1 min
+            setTimeout(inquire, 60000);
+        }
+    }
+
+    // create a notification
+    function inquire() {
+        // dimension7 is "helpfulness"
+        ga('set', 'dimension7', 'Yes');
```

Comment:
OK.  In that case, I suggest:

```
if(ga) ga('set', 'dimension7', 'Yes');
```

..since we don't use GA locally.  Threw an error for me.